views/components/contact-us-component.njk

```diff
@@ -0,0 +1,10 @@
+{% macro contactUsComp() %}
+    <div>
+    <hr class="contactus-line">
```

Comment:
if you add a class to the div then you can style that line without using a hr tag. Also color and border not matching invision design